packages-ts/gauntlet-terra-contracts/src/commands/contracts/cw4_group/deploy.ts

```diff
@@ -1,6 +1,7 @@
 import { CATEGORIES } from '../../../lib/constants'
 import { isValidAddress } from '../../../lib/utils'
-import { AbstractInstruction, instructionToCommand } from '../../abstract/executionWrapper'
+import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
```

Comment:
This is not used, right?

Reply:
Correct.  Originally, I was doing something similar to what you suggested in the other comment... which required importing `CONTRACT_LIST` everywhere.  After splitting it into `AbstractInstructionTemplate` and `AbstractInstruction`, that made it unnecessary to import `CONTRACT_LIST` in most of the files... must have missed this one while going through and removing all of the imports.  If we do decide to import `AbstractInstruction` directly from `@gauntlet-terra`, then we'll have to keep this import and add a lot more of them.